framework/auth/views.py

```diff
@@ -242,7 +242,7 @@ def register_user(**kwargs):
 
     """
     # Verify email address match
-    if request.json['email1'] != request.json['email2']:
+    if str(request.json['email1']).lower() != str(request.json['email2']).lower():
```

Comment:
`request.get_json()` should be used here; `request.json` is deprecated: http://flask.pocoo.org/docs/0.10/api/#flask.Request.json.

I will make this change when I merge.